lib/ancestry/instance_methods.rb

```diff
@@ -107,13 +102,9 @@ def has_parent?
 
     def ancestry_changed?
       column = self.ancestry_base_class.ancestry_column.to_s
-      if ActiveRecord::VERSION::STRING >= '5.1.0'
```

Comment:
maybe we want to add a rails 6 fix here?
per the comment below
that is probably a different pr